patterns/1-initial/principles.md

```diff
@@ -0,0 +1,152 @@
+## Title
+
+Explicit InnerSource Principles
+
+## Patlet
+
+InnerSource as a term is circulating among employees. The usual explanation of
+"applying open source best practices inside an organisation" does not work well
+though with people lacking an open source background. As a remedy the most
+important principles of InnerSource get published widely.
+
+## Problem
+
+The organisation is trying to roll out InnerSource at a larger scale. The
+initiative started among open source enthusiasts. The goal is now to get buy-in
+from people that are lacking open source experience. For that audience the typical
+slogan of "applying open source best practices" is no longer sufficient to
+transport the message of what InnerSource is, which problems it solves and which
+tools it uses for solving these issues. As a result InnerSource adoption in the
+organisation slows down. Teams develop diverging ideas of what the goals of InnerSource
+is about and how to best implement it leading to confusion when contributors
+are starting to cross team boundaries.
+
+## Story
+
+Early experiments in an organisation have shown that open source collaboration
+best practices can be beneficial. The goal now is to move the initiative to
+teams and individuals lacking a deep background in open source.
+
+The goal now is to clearly communicate the goals of the InnerSource initiative
+as well as a clear path towards achieving these goals.
+
+## Forces
```

Comment:
```suggestion
## Context

* InnerSource as a term is circulating among employees.
* The initiative started among open source enthusiasts. 

## Forces
```

I just realized that the pattern didn't provide a Context block yet.
The main purpose of that block is for readers to answer the question: "Do I have this same particular situation?"

I added a minimal version of such a Context, based on information in other parts of this pattern.
I would suggest to improve this further in future iterations of this pattern (but not important right now).